apps/apps.go

```diff
@@ -20,8 +20,10 @@ import (
 	"tidbyt.dev/community/apps/dateprogress"
 	"tidbyt.dev/community/apps/datetimeclock"
 	"tidbyt.dev/community/apps/daynightmap"
+	"tidbyt.dev/community/apps/destiny2stats"
```

Comment:
I think these might be causing some of the checks to fail - you should remove references to other apps from your PR